packages/oidc-js/src/models/client.ts

```diff
@@ -34,7 +34,9 @@ interface BaseConfigInterface {
     responseMode?: ResponseMode;
     scope?: string[];
     serverOrigin: string;
-    endpoints?: ServiceResourcesType;
+    endpoints?: OIDCEndpoints;
+    overrideWellEndpointConfig?: boolean;
```

Comment:
Shall we rename this to `overrideWellKnownEndpointConfig`?

May be we can do this in a separate PR.

Reply:
@thivi

Reply:
Tracked here: https://github.com/asgardio/asgardio-js-oidc-sdk/issues/59